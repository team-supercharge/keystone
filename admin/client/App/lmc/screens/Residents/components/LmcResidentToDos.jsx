import React, { Component } from 'react'
import { connect } from 'react-redux'

export class LmcResidentToDos extends Component {
    render() {
        return (
            <div>To-Dos</div>
        )
    }
}

const mapStateToProps = (state) => {

}

export default connect(mapStateToProps)(LmcResidentToDos)