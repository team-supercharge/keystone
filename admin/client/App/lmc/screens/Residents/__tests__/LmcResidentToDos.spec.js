import React from 'react'
import { shallow } from 'enzyme'
import { LmcResidentToDos } from '../components/LmcResidentToDos.jsx'

describe('LmcResidentToDos', () => {
    let wrapper

    beforeEach(() => {
        wrapper = shallow(
            <LmcResidentToDos />
        )
    })

    it('renders correctly', () => {
        expect(wrapper).toMatchSnapshot()
    })
})