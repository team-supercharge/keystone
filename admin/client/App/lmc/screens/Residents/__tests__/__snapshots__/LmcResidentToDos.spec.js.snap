// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LmcResidentToDos renders correctly 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div>
    To-Dos
  </div>,
  "nodes": Array [
    <div>
      To-Dos
    </div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <LmcResidentToDos />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": LmcResidentToDos {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div>
          To-Dos
        </div>,
        "_debugID": 2,
        "_renderedOutput": <div>
          To-Dos
        </div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <LmcResidentToDos />,
}
`;
